### ISSUE TITLE
1.20.2

### ISSUE THREAD

**Issue by Terraphice** [RuffleSteels/Firorize#26]
Building the 1.20.2 branch gives a .jar that causes some Mixin errors, and the branch seems very out of date in general compared to 1.20.1 & the master, so I'm assuming there's no real 1.20.2 support.

Obligatory backport request warning, please port your 1.20.1 final version to 1.20.2. I would do it, but you self-implemented a lot of things that I really don't have the time to try to comprehend with my limited Java experience.

**Comment by RuffleSteels** [comment]
yeah i've done a lot of tinkering with existing game code so i'll do this asap

**Comment by RuffleSteels** [comment]
I will not be updating to 1.20.2, I have 1.21.11 version 461918d90332b30fbccbc8c3cbb2ab3b89f3b777
